BroadcomCloudSwg/connectors/broadcom_cloud_swg_connector.py

```diff
@@ -200,23 +200,33 @@ async def get_events(self) -> tuple[int, datetime]:
 
         data_to_push: list[dict[str, str]] = []
         result = 0
-        latest_date: datetime | None = None
+        previous_latest_date: datetime = start_date
```

Comment:
I still believe we should shift the start date to the upper hour (according to [this documentation](https://techdocs.broadcom.com/us/en/symantec-security-software/web-and-network-security/cloud-swg/help/cloudswg-api-reference/report-sync-about/report-sync-api.html#topic.dita_4509cfa5-a73d-4ebc-8a12-9e398dc2e8f5_section_6)).
```suggestion
        previous_latest_date: datetime = (start_date.replace(minute=0, second=0,microsecond=0) + timedelta(hours=1))
```